sickbeard/helpers.py

```diff
@@ -114,65 +114,81 @@ def remove_non_release_groups(name, clean_proper=False):
     # Check your database for funky release_names and add them here, to improve failed handling, archiving, and history.
     # select release_name from tv_episodes WHERE LENGTH(release_name);
     # [eSc], [SSG], [GWC] are valid release groups for non-anime
+
+    # All lowercase and alphabetically sorted
+    groups = {
+        '-20-40',
+        '1044',
+        'agusiq',
+        'androidtwou',
```

Comment:
Instead of inserting groups into the regex you should just make a capturing group to capture anything between the delimiters, then check and see if the captured item is in the group list, if so then replace the entire tag.